libfwupdplugin/fu-bios-attrs.c

```diff
@@ -66,6 +66,9 @@ fu_bios_attr_set_description(FwupdBiosAttr *attr, GError **error)
 
 	g_return_val_if_fail(FWUPD_IS_BIOS_ATTR(attr), FALSE);
 
+	if (fwupd_bios_attr_get_kind(attr) == FWUPD_BIOS_ATTR_KIND_READ_ONLY)
+		return TRUE;
```

Comment:
not convinced about this one; feels like it's being done in the wrong layer.

Reply:
any proposal of which layer makes more sense?